src/main/java/com/sourcegraph/vcs/ConvertUtil.kt

```diff
@@ -2,13 +2,15 @@ package com.sourcegraph.vcs
 
 import java.net.URI
 
-fun convertGitCloneURLToCodebaseNameOrError(cloneURL: String): String {
+data class CodebaseName(val value: String)
```

Comment:
Is the `value` always some kind of the url?
If yes, maybe we can make the name more descriptive?
If no, maybe it warrants a comment what value we can expect there?

Reply:
I think `CodebaseName` is the description here

Reply:
Codebase name does not suggest url-like-string at all to me. 
And it's not defined anywhere what it really means.
Anyway, it's just my point of view, there is nothing wrong with the code per se.

Reply:
> Codebase name does not suggest url-like-string at all to me.

tbh I'd say that's a good sign - it will be harder to confuse it with URL 😅